### PR TITLE
add aaplus/2.41

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by jgsogo — DISMISSED** [review]

File: recipes/aaplus/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+required_conan_version = ">=1.33.0"
+
+
+class Aaplusconan(ConanFile):
+    name = "aaplus"
+    description = (
+        "AA+ is a C++ implementation for the algorithms as presented in the "
+        "book \"Astronomical Algorithms\" by Jean Meeus"
+    )
+    license = "?"
```
Comment (uilianries) [inline recipes/aaplus/all/conanfile.py:15]:
```suggestion
    license = "Unlicense"
```

https://spdx.org/licenses/Unlicense.html
The copyright listed on headers is pretty close to unlicense

**Review by uilianries — DISMISSED** [review]

File: recipes/aaplus/all/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/aaplus/all/conanfile.py:15]: correct, forgot to edit this field :)

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Thanks for be so active space, I am always able to check all of my bots features 24/7 ❤️

**Review by SSE4 — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**